docs/faq.md

```diff
@@ -4,6 +4,37 @@ title: 'Frequently Asked Questions'
 
 Here are some answers to frequently asked questions. If you have a question, you can ask it by opening an issue on the [Storybook Repository](https://github.com/storybookjs/storybook/).
 
+* [How can I opt-out of Angular Ivy?](#how-can-i-opt-out-of-angular-ivy)
+* [How can I opt-out of Angular ngcc?](#how-can-i-opt-out-of-angular-ngcc)
+* [How can I run coverage tests with Create React App and leave out stories?](#how-can-i-run-coverage-tests-with-create-react-app-and-leave-out-stories)
+* [I see `ReferenceError: React is not defined` when using Storybook with Next.js](#i-see-referenceerror-react-is-not-defined-when-using-storybook-with-next.js)
+* [How do I setup Storybook to share Webpack configuration with Next.js?](#how-do-i-setup-storybook-to-share-webpack-configuration-with-next.js)
```

Comment:
@kylemeenehan, these aren't working properly when booting up the website locally, they instead should be:
```md
- [I see `ReferenceError: React is not defined` when using Storybook with Next.js](#i-see-referenceerror-react-is-not-defined-when-using-storybook-with-nextjs)
- [How do I setup Storybook to share Webpack configuration with Next.js?](#how-do-i-setup-storybook-to-share-webpack-configuration-with-nextjs)
```

Reply:
Thanks for picking that up, and sorry I missed it!

I've made the changes and pushed.